styles.css

```diff
@@ -87,3 +87,249 @@ main ul {
 .frontpage {
   padding: 0 5%;
 }
+
+/* Works */
+
+.works {
+  display: grid;
+  justify-items: center;
+  margin-bottom: 30%;
+}
+
+.projimage {
+  display: flex;
+  justify-content: center;
+}
+
+.recent {
+  color: #1c1a19;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 25%;
+  border-bottom: 1px solid #212121;
+  padding-bottom: 10%;
+  margin-bottom: 15%;
+}
+
+.card {
+  padding-bottom: 15%;
```

Comment:
- [ ] As the parent `div`, this class is meant to be styled using `flexbox` to align the content. This is a major requirement which tests your use of this feature. Kindly adhere to this to nake your content alignment easy , and prove the skills you've learned which is valuable to your career 👍